index.php

```diff
@@ -1,36 +1 @@
-<?php
-/**
- * The main template file
- *
- * This is the most generic template file in a WordPress theme
- * and one of the two required files for a theme (the other being style.css).
- * It is used to display a page when nothing more specific matches a query.
- * E.g., it puts together the home page when no home.php file exists.
- *
- * @link https://developer.wordpress.org/themes/basics/template-hierarchy/
- *
- * @package Go
- */
-
-get_header();
-
-Go\page_title();
-
-if ( have_posts() ) {
-
-	// Start the Loop.
-	while ( have_posts() ) :
-		the_post();
-		get_template_part( 'partials/content' );
-	endwhile;
-
-	// Previous/next page navigation.
-	get_template_part( 'partials/pagination' );
-
-} else {
-
-	// If no content, include the "No posts found" template.
-	get_template_part( 'partials/content', 'none' );
-}
-
-get_footer();
+<?php // This page intentionally left blank.
```

Comment:
Seems like php linting does not like this comment. I guess we could always do something like 

```suggestion
<?php
/**
 * This page intentionally left blank.
 *
 * @package Go
 */
```

Reply:
I'd rather save the bytes and ignore linting the file.